lib/Controller/SignalingController.php

```diff
@@ -178,7 +172,6 @@ public function pullMessages($token) {
 				break;
 			}
 
-			$this->dbConnection->close();
```

Comment:
This should actually be kept, to free up connections with the database. Otherwise if you have more users then the defined limit of open database connections you have a problem.

Reply:
Yes, `$this->messages->getAndDeleteMessages($sessionId)` opens a connection and that is freed now, but with your patch its kept open, potentially running in a timeout too, when the db limit is lower than the loop time.

Reply:
Ah, OK, I see; I expected `Messages` to clean up after itself and I did not check the code, sorry. Then I will close this pull request.